internal/binutils/binutils.go

```diff
@@ -144,7 +144,9 @@ func initTools(b *binrep, config string) {
 		// that if the tool is not found by its default name.
 		b.addr2line, b.addr2lineFound = findExe("gaddr2line", append(paths["addr2line"], defaultPath...))
 	}
-	b.nm, b.nmFound = findExe("nm", append(paths["nm"], defaultPath...))
+	// The output of llvm-nm and GNU nm with "-n" option is interchangeable for
```

Comment:
The output is one thing. What about the flags? And version requirements? E.g. the flags and the output might be compatible between Binutils' nm and LLVM's llvm-nm starting certain versions but not for older ones?

Reply:
The version check can be omitted. `llvm-nm -n` has had similar output since 2011. There is no need checking older versions.

Reply:
Are their flags compatible too?

Reply:
https://github.com/llvm/llvm-project/commit/b8672a50c08dcd8b4805f4ca4b33354acc3c127c (2011) added "n". I think we don't need to worry about the output compatibility.

Reply:
Updated the comment here.